modules/cnv_install_1_4.adoc

```diff
@@ -0,0 +1,53 @@
+[[install_cnv_apb]]
+=== Installing {ProductName} to {product-title}
+
+The `kubevirt-ansible` RPM contains the latest automation for deploying 
+{ProductName}. This procedure installs all {ProductName} components to your 
+{product-title} cluster. 
+
+This procedure installs the following components:
+
+* {ProductName} core components (KubeVirt)
+* Containerized data importer (CDI) controller
+* Multus and Open vSwitch (OVS) container network interface plug-ins
+* Updated {product-title} web console
+
+.Prerequisites
+* {product-title} 3.11 cluster, running
+* User with `cluster-admin` privileges
+* xref:../apb_devel/cli_tooling.adoc#apb-devel-cli[APB tool]
```

Comment:
Great, I'll remove the APB tool from the prerequisites. 

@dankenigsberg - should I also remove "User with `cluster-admin` privileges"? @simon3z mentioned via email that I may be able to remove the entire section about creating a cluster admin user. This would leave the 3.11 cluster as the only prerequisite.

Reply:
I think that we *still* need a cluster admin in order to deploy, and we need to `oc login` so we have her credentials. I don't understand how we can drop it.

Reply:
We will still need to login as an admin in order to be able to create Cluster scoped objects.  So I think you can remove the apb sections, but replace it with what Dan mentioned: ```oc login ....``` or a doc line that says something like ```You need to login as admin```

Reply:
Adding the login command as the first step of the install procedure.